### ISSUE TITLE
cauldron

### ISSUE THREAD

**Issue by lipki** [SummerFields/Mod-Mintest-xDecor#11]
![xdecor_cauldron_sides](https://cloud.githubusercontent.com/assets/401965/15862580/67a2672c-2cd0-11e6-9d53-c3883eb76127.png)![xdecor_cauldron_top_anim_boiling_water](https://cloud.githubusercontent.com/assets/401965/15862581/67d1ea38-2cd0-11e6-92ca-68523d2ae6db.png)![xdecor_cauldron_top_anim_soup](https://cloud.githubusercontent.com/assets/401965/15862584/67e1deb6-2cd0-11e6-8623-65f97cddb113.png)![xdecor_cauldron_top_empty](https://cloud.githubusercontent.com/assets/401965/15862582/67e09ace-2cd0-11e6-81c4-0e3cc40b672b.png)![xdecor_cauldron_top_idle](https://cloud.githubusercontent.com/assets/401965/15862583/67e102ac-2cd0-11e6-84ae-e0aae0002694.png)

**Comment by lipki** [comment]
![xdecor_cauldron_sides](https://cloud.githubusercontent.com/assets/401965/15876960/11e6bdc4-2d11-11e6-95a2-195b5071d254.png)![xdecor_cauldron_top_anim_boiling_water](https://cloud.githubusercontent.com/assets/401965/15877009/66579248-2d11-11e6-98c1-7b07de2d47e5.png)![xdecor_cauldron_top_anim_soup](https://cloud.githubusercontent.com/assets/401965/15876963/11f3f020-2d11-11e6-82b7-85e587794b3c.png)![xdecor_cauldron_top_empty](https://cloud.githubusercontent.com/assets/401965/15876962/11f18966-2d11-11e6-9067-fe6b7cb80089.png)![xdecor_cauldron_top_idle](https://cloud.githubusercontent.com/assets/401965/15876964/11f56d06-2d11-11e6-92e9-33286109460d.png)

I have not tested the boiling water and soup.
I do not know the recipe.

**Comment by tobyplowy** [comment]
I'm going to see if I can find the recipe

**Comment by tobyplowy** [comment]
Well I know how it works now it took me some time but yeah (here is how you do it place a Cauldron and grab a water bucket right click on the Cauldron (to fill it up with water) then put fire underneath the Cauldron and then just simply throw food items in it (I used an apple and a brown mushroom)

**Comment by tobyplowy** [comment]
this is what it looks like
![olollghkb tu](https://cloud.githubusercontent.com/assets/16853304/15892949/06f8f67c-2d7d-11e6-9c7d-efd9c8358b21.png)

**Comment by lipki** [comment]
3d models are so ugly

**Comment by tobyplowy** [comment]
Yeah they should have made actual 3D models (there too lazy I guess) also would you be interested in making something for minetest (making a summerfields sup game) that would be awesome

**Comment by lipki** [comment]
I do not understand

**Comment by tobyplowy** [comment]
Do you want to make a Mod for minetest?

**Comment by tobyplowy** [comment]
A sup game simply said it's a (Mod pack) for minetest

**Comment by tobyplowy** [comment]
Sorry didn't mean to close the issue

**Comment by lipki** [comment]
OK.
I wanted to do modpack for my Minecraft team  (farmcraft.info)
But I think mainly take an existing thing.
I do not know Minetest ...

**Comment by tobyplowy** [comment]
That would be amazing if you are interested in making mods for minetest you can use [this](http://rubenwardy.com/minetest_modding_book) for help even I could mod some basic stuff (new blocks And items) and I can't program :/ (a little hint you can copy paste some code, but don't tell anyone yes that's also how you make the items and blocks :wink:)
